assets/src/scss/layout/_featured-action.scss

```diff
@@ -0,0 +1,94 @@
+.featured-action-section {
+  margin-top: 50px;
+  margin-bottom: 50px;
+  display: flex;
+  border-radius: 4px;
+  gap: 24px;
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.16);
```

Comment:
Maybe we should follow up with the @greenpeace/planet-4-designers if we can use a token color instead of `rgba(0, 0, 0, 0.16)`!